### PR TITLE
Add sampling rate to SetOfCoordinates3D

### PR DESCRIPTION
**Pull request by EstrellaFG**
-Attribute **samplingRate** added to object `SetOfCoordinates3D` with the corresponding get and set.
-In `protocol_import_coordinates` set samplingRate of object SetOfCoordinates3D

This changes are required for the correct working of `protocol_tomo_extraction` in `scipion-em-eman2` after PR scipion-em/scipion-em-eman2/pull/33

*Extra: **Remove non-tomography imports** from `protocols.conf` by user request (they have no sense in tomography menu of scipion and they are confusing for the user).

### CODE REVIEW

File: tomo/objects.py [hunk 1]
```diff
@@ -463,6 +463,7 @@ def __init__(self, **kwargs):
         data.EMSet.__init__(self, **kwargs)
         self._volumesPointer = pwobj.Pointer()
         self._boxSize = pwobj.Integer()
+        self._samplingRate = pwobj.Integer()
```
Comment (adrianq) [inline tomo/objects.py]: @EstrellaFG Maybe a float?

File: tomo/protocols.conf [hunk 2]
```diff
@@ -5,12 +5,8 @@ Tomography = [
 	{"tag": "protocol", "value": "ProtImportTomograms",      "text": "default"},
 	{"tag": "protocol", "value": "ProtImportSubTomograms",      "text": "default"},
 	{"tag": "protocol", "value": "ProtImportCoordinates3D",      "text": "default"},
-	{"tag": "protocol", "value": "ProtImportMicrographs", "text": "import micrographs"},
-	{"tag": "protocol", "value": "ProtImportParticles",   "text": "import particles"},
 	{"tag": "protocol", "value": "ProtImportVolumes",     "text": "import volumes"},
 	{"tag": "section", "text": "more", "openItem": "False", "children": [
-	{"tag": "protocol", "value": "ProtImportCoordinates",   "text": "import coordinates"},
-	{"tag": "protocol", "value": "ProtImportCTF",   "text": "import ctfs"},
 	{"tag": "protocol", "value": "ProtImportPdb",         "text": "import atom structure"},
```
Comment (adrianq) [inline tomo/protocols.conf]: @EstrellaFG Does it make sense to give these other 4 imports? Just asking...

Comment (EstrellaFG) [reply tomo/protocols.conf]: I have talk with users and we have decided to remove also "`ProtImportPdb`" but import volume, average and mask seems to be useful in tomography (even if they are 2D).  However, we have place those three imports in "`more`".

File: tomo/objects.py (hunk 1 above)
Comment (EstrellaFG) [reply tomo/objects.py]: Sure

**Comment by EstrellaFG** [comment]
> @EstrellaFG It looks good in general. I have added some minor comments/questions below. Also, would you mind modifying the import set of coordinates test to check these new parameters? Thanks!

All requested changes in https://github.com/scipion-em/scipion-em-tomo/pull/18/commits/26d5d0dbacb59a95a85838a16661a95276b6b814

**Pull request merged**